www/assessment/customization/style-guide/jest.config.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path');

```

Comment:
Remove this file, I am not sure we will even have unit-tests in this first phase